conda/recipes/cudf/meta.yaml

```
@@ -2,9 +2,9 @@
 
 {% set version = environ.get('GIT_DESCRIBE_TAG', '0.0.0.dev').lstrip('v') + environ.get('VERSION_SUFFIX', '') %}
 {% set minor_version = version.split('.')[0] + '.' + version.split('.')[1] %}
-{% set py_version=environ.get('CONDA_PY', 38) %}
-{% set cuda_version='.'.join(environ.get('RAPIDS_CUDA_VERSION', '11.5.2').split('.')[:2]) %}
-{% set cuda_major=cuda_version.split('.')[0] %}
+{% set py_version = environ.get('CONDA_PY', 38) %}
```

should we also change this to be required? seems pretty disastrous to accidentally publish a conda package with the wrong Python version in the build string.

```suggestion
{% set py_version = environ['CONDA_PY'] %}
```

---

(also relevant to other files in this PR)

---

I’m not sure exactly how this works. I think this variable is provided by conda and not the environment. https://docs.conda.io/projects/conda-build/user-guide/environment-variables.html

I’m open to the change you proposed, I just don’t know if there are any catches we are unaware of.